snap/revision.go

```diff
@@ -44,6 +44,10 @@ func (r Revision) Unset() bool {
 	return r.N == 0
 }
 
+func (r Revision) Set() bool {
```

Comment:
as we have already quite a bit of code using !Unset, I'm not entirely sure if this addition is worth it at this time